symbols/arm9.yml

```diff
@@ -3394,8 +3404,6 @@ arm9:
         r0: move
         return: bool
     - name: IsUsableWhileTaunted
-      aliases:
-        - IsAffectedByTaunt
```

Comment:
I'd rather keep this one even though it's wrong. It's not an internal system function so it's more likely someone could be using it.